.devcontainer/Dockerfile

```
@@ -1,4 +1,4 @@
-ARG VARIANT=1.23-bookworm
+ARG VARIANT=1.24.1-bookworm
```

This tag doesn't exist.
```suggestion
ARG VARIANT=1.24-bookworm
```